py/test/selenium/webdriver/common/bidi_network_tests.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from selenium import webdriver
+from selenium.webdriver.common.bidi.network import Network
+
+
+def url_for(page):
+    return webserver.where_is(page)
+
+@pytest.fixture
+def driver():
```

Comment:
There are some existing fixtures we can use for these tests, please see an example of usage in https://github.com/SeleniumHQ/selenium/blob/d9739391458e315d2dbff99dcf068444bde5db61/py/test/selenium/webdriver/common/bidi_script_tests.py

Reply:
You're right